com.chartboost.mediation/Android~/build.gradle

```diff
@@ -22,7 +22,7 @@ ext {
     privatePass = onlyPublic? "" : System.getenv("JFROG_PASS")
 
     inputVersionIsHigher = true
-    chartboostMediationVersion = '4.+'
+    chartboostMediationVersion = '2023.9.6.1693990896-nightly'
```

Comment:
🤔 don't think we want to use this version for the demo.

Reply:
Oh yeah that got pushed by mistake. I'll remove it